db/schema.rb

```diff
@@ -271,6 +266,14 @@
 
   add_index "content_buckets_homerooms", ["content_bucket_id", "homeroom_id"], name: "content_bucket_homeroom_index", unique: true, using: :btree
 
+  create_table "content_buckets_purchases", id: false, force: true do |t|
```

Comment:
oh my bad I'll remove it from that branch and then merge it in here then I guess